client/src/components/Leaflet.js

```diff
@@ -250,20 +277,94 @@ const Leaflet = ({
     widthPropUnchanged
   ])
 
+  useEffect(() => {
+    if (!map || !layerControl || !allLocations?.length) {
+      return
+    }
+    const allMarkers = allLocations
+      .filter(loc => Location.hasCoordinates(loc))
+      .map(location => {
+        const popupContent = document.createElement("div")
+        popupContent.setAttribute("style", "width: 300px;text-align: center")
+
+        return new Marker([location.lat, location.lng], {
+          icon: locationIcon,
+          draggable: false,
+          autoPan: false,
+          id: location.uuid
+        })
+          .bindTooltip(location.name, {
+            direction: "top",
+            permanent: true,
+            offset: [0, -58]
+          })
+          .bindPopup(popupContent)
+          .on("popupopen", e => {
+            ReactDOM.render(
+              <>
+                <b>
+                  {/* // TODO: Find a way to work with LinkTo in different entry point here, not nested inside Router context */}
+                  <a href={`/locations/${location.uuid}`}>{location.name}</a>
```

Comment:
I had this problem when trying to use LinkTo, any suggestions are welcome!

Reply:
@ysf-simsoft @cemalettin-simsoft Maybe try https://react-leaflet.js.org/ ?

Reply:
I think that would be a big refactor, since all of the maps are implemented by original leaflet library. We can open a dept issue on that.
I will just remove the link and simply render its name for workaround.